### PR TITLE
Added case for MINGW64_NT-6.1 in SEDCMD() function to support GitBash…

### PR DESCRIPTION
**Pull request by skryukova**
… on Windows

*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

### CODE REVIEW

File: utilities/add_new_event.sh [hunk 1]
```diff
@@ -13,6 +13,8 @@ SEDCMD(){
     sed -i "$@"
   elif [[ $OSNAME == 'Darwin' && $GNUSED == '/usr/local/bin/sed' ]]; then
     sed -i "$@"
+  elif [[ $OSNAME == 'MINGW64_NT-6.1' ]]; then
```
Comment (MikeRosTX) [inline utilities/add_new_event.sh]: At a simple glance, the change for all the files looking fine.
I'll give a test run tomorrow 04/29/19.

**Comment by kmugrage** [comment]
I haven't attempted any troubleshooting yet...


```
Ken Mugrage@DESKTOP-6FPF9B1 MINGW64 ~/Dev/devopsdays-web/utilities (skryukova-master)                                                          $ ./add_new_event.sh                                                                                                                           Enter your event year (default: 2019): 2020                                                                                                    Enter your city name: Seattle                                                                                                                  Enter your devopsdays event twitter handle (defaults to devopsdays): devopsdayssea                                                             sed: can't read s/YYYY/2020/: No such file or directory
```
--

**Comment by kmugrage** [comment]
Ah, mine is `MINGW64_NT-10.0`
--

**Comment by kmugrage** [comment]
It seems like this would be better if the version of Windows wasn't so specific. I'm not sure how wide the match can be however.

File: utilities/add_new_event.sh (hunk 1 above)
Comment (kmugrage) [reply utilities/add_new_event.sh]: If I change this to 

`elif [[ $OSNAME == MINGW64* ]]; then`

then it works on MINGW64_NT-10.0

I don't have any other versions of WIndows to test

**Comment by skryukova** [comment]
@kmugrage I agree that your change to MINGW64* is better, would you like me to make this change, test it and redo the pull request?

**Comment by skryukova** [comment]
@kmugrage I've updated the match string and tested the scripts on my Windows 7 machine

**Comment by kmugrage** [comment]
Now I'm troubleshooting `cp -r examples/content/events/yyyy-city ../content/events/$event_slug`

On my machine this is creating `../content/events/yyyy-city/2020-seattle`

**Comment by skryukova** [comment]
I may be able to test on Windows 10 later. It works just fine on my Windows 7.

**Comment by skryukova** [comment]
I am going to close this, and keep working around it for now. I may try again later.

**Pull request closed without merging**